pubspec.yaml

```diff
@@ -38,7 +38,7 @@ dev_dependencies:
   flutter_driver:
     sdk: flutter
   args: ^2.3.1
-  flutter_lints: ^2.0.1
+  flutter_lints: ">=2.0.1 <4.0.0"
```

Comment:
@godofredoc was there a dependabot setting to prefer upgrading over widening dependency versions?